src/services/sanity.js

```diff
@@ -831,6 +831,22 @@ export async function fetchRelatedLessons(railContentId, brand) {
   return fetchSanity(query, false);
 }
 
+/**
+* Fetch all packs.
+* @param {string} brand - The brand for which to fetch packs.
+* @param {string} [searchTerm=""] - The search term to filter packs.
+* @param {string} [sort="-published_on"] - The field to sort the packs by.
+* @returns {Promise<Array<Object>|null>} - The fetched pack content data or null if not found.
+*/
+export async function fetchAllPacks(brand, sort = "-published_on", searchTerm = "") {
+  const sortOrder = getSortOrder(sort);
+
+  const query = `*[_type == 'pack' && brand == '${brand}' && title match "${searchTerm}*"]{
```

Comment:
Just FYI. 
I was curious if adding the search `title match "*"` would have any performance effect and it actually improved my test queries. I'm not sure what's going on here, but happy that sanity.